vllm/v1/request.py

```diff
@@ -159,8 +160,8 @@ def get_finished_reason(status: "RequestStatus") -> Union[str, None]:
 # are longer than the model's length cap. Therefore, the stop
 # reason should also be "length" as in OpenAI API.
 _FINISHED_REASON_MAP = {
-    RequestStatus.FINISHED_STOPPED: "stop",
-    RequestStatus.FINISHED_LENGTH_CAPPED: "length",
-    RequestStatus.FINISHED_ABORTED: "abort",
-    RequestStatus.FINISHED_IGNORED: "length",
+    RequestStatus.FINISHED_STOPPED: RequestFinishedReason.STOP,
```

Comment:
Is this map needed anymore now that we have an enum?

Reply:
Well, it's mapping one enum to another?

```
class RequestStatus(enum.IntEnum):
    ...
    FINISHED_STOPPED = 3
    FINISHED_LENGTH_CAPPED = 4
    FINISHED_ABORTED = 5
    FINISHED_IGNORED = 6
    
class RequestFinishedReason(enum.IntEnum):
    STOP = 0
    LENGTH = 1
    ABORT = 2
```

Reply:
I mean maybe we could just use RequestStatus everywhere?

Reply:
all good, I didnt realize that